pyrsa/model/model.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Definition of RSA Model class and subclasses
+@author: jdiedrichsen, heiko
+"""
+
+import numpy as np
+
+
+class Model:
+    """
+    Abstract model class.
+    Defines members that every class needs to have, but does not implement any
+    interesting behavior. Inherit from this class to define specific model
+    types
+    """
+    def __init__(self, name):
+        self.name = name
+        self.n_param = 0
+
+    def predict(self, theta=None):
+        """ Returns the predicted rdm(-vector)
+
+        Args:
+            theta(np.array 1d): the model parameter vector
```

Comment:
the precise name of the argument type here would be `numpy.ndarray`. If we use those, in the documentation it will generate a link to that class. The dimension expectations could then be in the description part